docs/docs/development/1_abstractions.md

```diff
@@ -515,6 +724,54 @@ When traveler leaves: 60 tourists are in the city c3
 ```
 `reset_tourists_1` executes when the `walker traveler` leaves the `city` node.
 
+### Here and Visitor
+
```

Comment:
Is it okay to have this section under abilities? @ypkang I first thought I should mention these before explaining the node abilities, but I felt like it's over-explaining there. So just kept a node.